dev_cmds/setup-dev.ts

```diff
@@ -0,0 +1,52 @@
+import { execPiped, runAsyncMain } from "devcmd";
+import { YARN_COMMAND } from "./utils/commands";
+import { devcmdPackageDir, devPath, singlePackageJsonExampleDir } from "./utils/paths";
+import fs from "fs-extra";
+
+async function main() {
+
+  if(await fs.pathExists(devPath)) {
+    process.stderr.write('A development folder already exists!\n')
+    process.exit(1)
+  }
+
+  await fs.copy(
+    singlePackageJsonExampleDir,
+    devPath,
+    {
+      // Only copy those files, for which the filter evaluates to true
+      filter: (src) => !src.includes('node_modules')
+    }
+  );
```

Comment:
is there a reason why this isn't part of the try-catch block below?

Reply:
Moving it :)